app/assets/javascripts/groups_index.js

```diff
@@ -0,0 +1,26 @@
+$(document).ready(function(){
+  var selectKind = $('.selectKind').val();
+  var selectNeighborhood = $('.selectNeighborhoods').val();
+
+  function groupList(selKind, selNeighborhood) {
+    $('tr').show();
+    if (selKind != 'all' && selNeighborhood != 'all') {
+      $('tr').not('.' + selNeighborhood).hide();
+      $('tr').not('.' + selKind).hide();
+    } else if (selectKind == 'all' && selNeighborhood != 'all') {
+      $('tr').not('.' + selNeighborhood).hide();
+    } else if (selNeighborhood == 'all' && selKind != 'all') {
```

Comment:
Super super nitpick, but I'd switch the order of the bools so it matches above, like: `else if (selKind != 'all' && selNeighborhood == 'all')`